scout/adapter/mongo/case.py

```diff
@@ -506,6 +506,9 @@ def load_case(self, config_data, update=False):
             LOG.warning(error)
 
         if existing_case and update:
+            if case_obj['status'] == 'active':
+                case_obj['status'] = 'inactive'
+                case_obj['research_requested'] = False
```

Comment:
Exactly! Only I think both should also be done if the case has status archived.

Reply:
If the case is archived it will be inactivated anyway, because of line 573 in adapter.case.py

Reply:
Yes, but not have its rerun flag lowered. Which I'm guessing you planned to not always set to false on update now?

Reply:
Sorry for second guessing you on this - should I make a commit instead? 😊
You are just much faster coming up with a suggestion - I get stuck in potential implications and possible other things that it touches.. 🙄

Reply:
> Yes, but not have its rerun flag lowered. Which I'm guessing you planned to not always set to false on update now?

I'm just thinking that it shouldn't be necessary to specify `case_obj['research_requested'] = False` because there is some code somewhere that does it already

Reply:
Can I try locally, it's quite easy to test it

Reply:
Oh, and I'm reading to sloppily: I was going about ```case_obj['rerun_requested']```.

Reply:
I'm also a bit confused, I think testing would be the best way to understand how all this works :-/

Reply:
```'research_requested'``` is already read from the case obj, so that should (probably) work. I'm now leaning towards that there should be a second function for just updating the case individual parameters without touching the status etc. But still, the load function should be responsible for setting case_obj['request_rerun'] to false, rather than assuming that was why the case was updated. 😬

Reply:
feel free to make commits if you find a solution!

Reply:
I think I will find lunch first. Just leave it when you feel its time to drop out and I'll take it from there if there is anything left to do. Multiple parallell commits are too complicated. 😅

Reply:
You are right, the rerun_requested in adapter.update_case will cause a bug and has to be fixed!